CIP-1694/README.md

```diff
@@ -1068,14 +1046,13 @@ The **technical group** consists of:
 * proportion of collateral needed for scripts (`collateralPercentage`)
 
 The **governance group** consists of all the new protocol parameters that are introduced in this CIP:
-* governance voting thresholds ($P_1$, $P_{2a}$, $P_{2b}$, $P_3$, $P_4$, $P_{5a}$, $P_{5b}$, $P_{5c}$, $P_6$, $P_7$, $Q_1$, $Q_{2b}$, $Q_4$)
-* constitutional committee term limits
-* governance action expiration
+* governance voting thresholds ($P_1$, $P_{2a}$, $P_{2b}$, $P_3$, $P_4$, $P_{5a}$, $P_{5b}$, $P_{5c}$, $P_{5d}$, $P_6$, $Q_1$, $Q_{2a}$, $Q_{2b}$, $Q_4$)
+* governance action maximum lifetime in epochs (`govActionLifetime`)
 * governance action deposit (`govDeposit`)
 * DRep deposit amount (`drepDeposit`)
-* DRep activity period (`drepActivity`)
-* minimal constitutional committee size
-* maximum term limit (in epochs) of the constitutional committee
+* DRep activity period in epochs (`drepActivity`)
+* minimal constitutional committee size (`ccMinSize`)
+* maximum term length (in epochs) of the constitutional committee (`ccMaxTermLength`)
```

Comment:
this sounds like its max term length for the entire committee, not individual terms